lib/linguist/languages.yml

```diff
@@ -1244,6 +1244,16 @@ Emacs Lisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 102
+Email:
+  type: data
+  extensions:
+  - ".eml"
+  - ".email"
+  - ".mail"
```

Comment:
Same for `.mail` as for `.email`: https://github.com/search?p=99&q=NOT+reply-to+extension%3Amail+NOT+from&type=Code